### PR TITLE
JAX-RS support for LocalDate, LocalDateTime and ZonedDateTime.

### PR DESCRIPTION
**Pull request by JanDoerrenhaus**

### CODE REVIEW

File: technology/jackson/src/main/java/com/github/mizool/technology/jackson/LocalDateSerializer.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import java.time.format.DateTimeFormatter;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+
+public class LocalDateSerializer extends JsonSerializer<LocalDate>
+{
+    @Override
+    public void serialize(LocalDate value, JsonGenerator gen, SerializerProvider serializers) throws IOException
```
Comment (bannmann) [inline technology/jackson/src/main/java/com/github/mizool/technology/jackson/LocalDateSerializer.java]: rename `gen` to `generator`

File: technology/jackson/src/main/java/com/github/mizool/technology/jackson/LocalDateTimeParamConverterProvider.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
+package com.github.mizool.technology.jackson;
```
Comment (bannmann) [inline technology/jackson/src/main/java/com/github/mizool/technology/jackson/LocalDateTimeParamConverterProvider.java]: add license boilerplate

File: technology/jackson/src/main/java/com/github/mizool/technology/jackson/LocalDateTimeParamConverterProvider.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+public class LocalDateTimeParamConverterProvider implements ParamConverterProvider
+{
+    @Override
+    public <T> ParamConverter<T> getConverter(Class<T> rawType, Type genericType, Annotation[] annotations)
+    {
+        ParamConverter<T> result = null;
+
+        if (rawType.equals(LocalDateTime.class))
+        {
+            result = new ParamConverter<T>()
```
Comment (bannmann) [inline technology/jackson/src/main/java/com/github/mizool/technology/jackson/LocalDateTimeParamConverterProvider.java]: This looks nearly identical to the `ParamConverter` inside the other two provider. Can we instead have a package local class that has a `DateTimeFormatter` constructor argument and is shared between the three providers?

File: technology/jackson/src/main/java/com/github/mizool/technology/jackson/ZonedDateTimeParamConverterProvider.java [hunk 4]
```diff
@@ -0,0 +1,51 @@
+package com.github.mizool.technology.jackson;
```
Comment (bannmann) [inline technology/jackson/src/main/java/com/github/mizool/technology/jackson/ZonedDateTimeParamConverterProvider.java]: license boilerplate

File: technology/jackson/src/main/java/com/github/mizool/technology/jackson/LocalDateTimeParamConverterProvider.java (hunk 3 above)
Comment (JanDoerrenhaus) [reply technology/jackson/src/main/java/com/github/mizool/technology/jackson/LocalDateTimeParamConverterProvider.java]: I don't see how, given the "ZonedDateTime.from" (etc) calls...

**Pull request merged**